web/docs/language/operator-modifiers.md

```diff
@@ -8,17 +8,73 @@ Operator modifiers are keywords that may occur before an operator.
 
 ## Scheduled Executions
 
-The special keyword `every` enables scheduled execution of an operator.
+### `every`
+The special keyword `every` enables scheduled execution of an operator at a fixed interval.
 
 Use the operator modifier like this:
 
 ```
 every <interval> <operator> [<args...>]
 ```
 
+#### Example
+
 For example, `version` prints the version number exactly once, but `every 1s
 version` prints the version number once every second.
 
+### `cron`
+The special keyword `cron` enables scheduled execution of an operator according to a [cron expression](https://en.wikipedia.org/wiki/Cron).
+
+Use the operator modifier like this:
+
+```
+cron "<cron expression>" <operator> [<args...>]
+```
+
+**Note that the `<cron expression>` must be enclosed in quotes.**
+
+
+A `<cron expression>`  has the form
+```
+<seconds> <minutes> <hours> <days of month> <months> <days of week>
+```
+where the 6 fields are separated by a space. 
+
+The field values can be as follows:
+
+| Field | Required | Allowed value * | Allowed special characters | Alternative Literals |
+| --- |--- | ---  | --- | --- |
+| seconds | yes | 0-59 | `*` `,` `-` | |
+| minutes | yes | 0-59 | `*` `,` `-` | |
+| hours | yes | 0-23 | `*` `,` `-` | |
+| days of month | 1-31 | 1-31 | `*` `,` `-` `?` `L` `W` | |
+| months | yes | 1-12 | `*` `,` `-` | `JAN` `FEB` `MAR` `APR` `MAY` `JUN` `JUL` `AUG` `SEP` `OCT` `NOV` `DEC` |
+| days of week | yes | 0-6 | `*` `,` `-` `?` `L` `#` | `SUN` `MON` `TUE` `WED` `THU` `FRI` `SAT` |
+
+\* - As described on Wikipedia [Cron](https://en.wikipedia.org/wiki/Cron)
+
+The special characters have the following meaning:
```

Comment:
You might have already figured it out, but just in case: You can use `cd web && yarn && yarn docusaurus start` to preview this.

The table does not look too nice when it's rendered in our docs. Below is a suggestion that simplifies it, and makes it look nicer.

```suggestion
The field values can be as follows:

| Field | Value | Special characters | Alternative literals |
| --- | ---  | --- | --- |
| seconds | 0-59 | `*` `,` `-` | |
| minutes | 0-59 | `*` `,` `-` | |
| hours | 0-23 | `*` `,` `-` | |
| days of month | 1-31 | `*` `,` `-` `?` `L` `W` | |
| months | 1-12 | `*` `,` `-` | `JAN` `FEB` `MAR` `APR` `MAY` `JUN` `JUL` `AUG` `SEP` `OCT` `NOV` `DEC` |
| days of week  | 0-6 | `*` `,` `-` `?` `L` `#` | `SUN` `MON` `TUE` `WED` `THU` `FRI` `SAT` |

The special characters have the following meaning:
```